tripy/tripy/frontend/module/module.py

```diff
@@ -67,6 +68,15 @@ class Module:
             "layernorm": tp.LayerNorm(2),
         }
 
+    This is another valid example with a wrapped :class:`tripy.avgpool` lambda function
+    ::
+
+        self.conv = tp.Conv(in_channels=2, out_channels=2, kernel_dims=(1,1), stride=(1,1))
```

Comment:
Can we either inline this or not make it a member? I think that would mess up `named_children()`

Reply:
Oh good point, may bad there. Fixed now